### ISSUE TITLE
Just uncovered a pretty concerning issue with VAMC Facility Health Services. There doesn't seem to be any logic preventing editors from mixing and matching Facilities, Systems, and Sections.

### ISSUE THREAD

**Issue by omahane** [department-of-veterans-affairs/va.gov-cms#11980]
Just uncovered a pretty concerning issue with VAMC Facility Health Services. There doesn't seem to be any logic preventing editors from mixing and matching Facilities, Systems, and Sections. 

I was able to save a Facility Health Service on staging as nickolas.sanchez@va.gov in a variety of mismatched ways

![Screen Shot 2022-12-15 at 4 33 39 PM](https://user-images.githubusercontent.com/51967950/207982001-ee09f465-7b39-47aa-b5ef-5859907225db.png)

![Screen Shot 2022-12-15 at 4 34 26 PM](https://user-images.githubusercontent.com/51967950/207982036-cf30edf5-be6f-4348-b3d6-06c3fe5bf1cd.png)

![Screen Shot 2022-12-15 at 4 36 43 PM](https://user-images.githubusercontent.com/51967950/207982061-6bf9d216-9fd5-4956-9224-02542626f2fb.png)

_Originally posted by @davidmpickett in https://github.com/department-of-veterans-affairs/va.gov-cms/issues/11026#issuecomment-1353806083_

**Comment by JQuaggles** [comment]
@omahane Does this fix the issues just for Lovell or for all health care sites? 
ie: https://github.com/department-of-veterans-affairs/va.gov-cms/issues/3545

**Comment by swirtSJW** [comment]
This did come up in discussion early on before we were versed on how services for lovell were going to work and that we have separate tricare specific text on the service tazonomy now.

    This shoud just be a JS alteration not a true validation.

**Comment by swirtSJW** [comment]
> @omahane Does this fix the issues just for Lovell or for all health care sites? ie: #3545

@JQuaggles It is a non-issue for other facility health services 
![image](https://user-images.githubusercontent.com/5752113/208126192-c312d683-583a-4a69-adbb-93b31c42b905.png)

![image](https://user-images.githubusercontent.com/5752113/208126226-9804e3b1-6a1d-4d26-8261-3564a65fce7a.png)

**Comment by omahane** [comment]
@davidmpickett I added ACs, but have a question about the 3rd, in which I left a blank to be filled in by you (I hope).

**Comment by omahane** [comment]
@davidmpickett @swirtSJW  On the VAMC System Health Service page, should we also remove "Lovell Federal health care" from the VAMC System dropdown?
<img width="1840" alt="Screen Shot 2022-12-20 at 8 02 38 AM" src="https://user-images.githubusercontent.com/766573/208685514-4c81eb5d-4bbd-4ae2-ae9d-dee1e145dfeb.png">

**Comment by swirtSJW** [comment]
> @davidmpickett @swirtSJW On the VAMC System Health Service page, should we also remove "Lovell Federal health care" from the VAMC System dropdown? 


Yes please.  Sorry I thought we had that so it winnowed based on section selection.    But given section comes last in that, it doesn't make much sense.

**Comment by omahane** [comment]
The work for the winnower appears to be done. That is, when adding or editing a VAMC System Health Service, VAMC Facility Service or a VAMC Facility Non-clinical Service as a "content_admin," the appropriate the sections are available and triggering the winnowing of the VAMC system health service and Facility dropdowns. 

However, when creating or editing as a "vamc_content_creator," the Lovell options are not available in the VAMC system health service and Facility dropdowns, and thus cannot be winnowed.

**Comment by omahane** [comment]
Today at mid-week check-in, @swirtSJW mentioned that no editor will not have the "content_publisher" role, which I wasn't setting with the Cypress tests. After manually creating a user with both the "vamc_content_creator" and "content_publisher" role, I can confirm that the dropdowns for Lovell are now working correctly. Steve said that he'll work on getting a Cypress step function in place to create a user with multiple roles. Currently, there's no way to do this in Cypress.
